llvm/lib/Transforms/Instrumentation/RealtimeSanitizer.cpp

```diff
@@ -17,47 +17,77 @@
 #include "llvm/IR/IRBuilder.h"
 #include "llvm/IR/Module.h"
 
+#include "llvm/Demangle/Demangle.h"
 #include "llvm/Transforms/Instrumentation/RealtimeSanitizer.h"
 
+#include <vector>
+
 using namespace llvm;
 
+static std::vector<Type *> getArgTypes(ArrayRef<Value *> FunctionArgs) {
+  std::vector<Type *> Types;
+  for (Value *Arg : FunctionArgs)
```

Comment:
Could reserve here before the for loop to prevent multiple resizing

Reply:
Done